roles/uperf-bench/templates/workload_store.yml.j2

```diff
@@ -12,14 +12,19 @@ spec:
       labels:
         app: uperf-bench-client
     spec:
+{% if uperf.hostnetwork %}
+      hostNetwork: true
+      serviceAccountName: benchmark-operator
+      serviceAccount: benchmark-operator
+{% endif %}
       containers:
       - name: benchmark
         image: "quay.io/benchmark-operator/uperf:latest"
         command: ["/bin/sh", "-c"]
         args:
           - "cat /tmp/uperf-test/uperfXML;
              export h={{item.status.podIP}};
-             uperf -v -x -a -m /tmp/uperf-test/uperfXML -i 1 | tee {{results.path}}/$(date +%s)-uperf"
+             seq {{uperf.rerun}} | xargs -I -- uperf -v -x -a -m /tmp/uperf-test/uperfXML -i 1 | tee {{results.path}}/$(date +%s)-uperf"
```

Comment:
I don't like creating new results such that there's no concrete manner to link the reruns, but this seems to be out of scope of this PR. This should definitely be addressed when we retrospect on indexing results.